spec/controllers/auctions_controller_spec.rb

```diff
@@ -9,6 +9,45 @@
       expect(auction).to be_a(AuctionViewModel)
       expect(auction.id).to eq(auction_record.id)
     end
+
+    context 'the list of auctions is sorted' do
+      it 'renders them descending by datetime' do
+
+        Timecop.freeze
+
+        date_start = Time.now - 3.days
+        date_latest = Time.now + 3.days
+        date_middle = Time.now + 2.days
+        date_first = Time.now + 1.days
+
+        auction_record_1 = create(
+          :auction,
+          start_datetime: date_start,
+          end_datetime: date_middle)
+
+        auction_record_2 = create(
+          :auction,
+          start_datetime: date_start,
+          end_datetime: date_latest)
+
+        auction_record_3 = create(
+          :auction,
+          start_datetime: date_start,
+          end_datetime: date_first)
+
+        get :index
+        auctions = assigns(:view_model).auctions
+
+        auction_1 = auctions[0]
+        auction_2 = auctions[1]
+        auction_3 = auctions[2]
+
+        expect(auction_1).to be_a(AuctionViewModel)
+        expect(auction_1.end_datetime).to eq(date_latest)
```

Comment:
do you mean just check to make sure that the auction records equal each other instead of one of their underlying attributes? Like this:

`expect(auction_1).to eq(auction_record_2)`